### PR TITLE
Issue/838 user refactor

### PR DESCRIPTION
**Pull request by kardan**
- [ ] Update release notes
- [ ] Test plan | Unit test | Integration test
- [ ] Copyright header
- [ ] Code formatting
- [ ] Documentation

### CODE REVIEW

**Comment by kardan** [comment]
@jonase added Selmer to render email templates

File: backend/src/akvo/lumen/endpoint/invite.clj [hunk 1]
```diff
@@ -1,28 +1,40 @@
...
-            [akvo.lumen.component.user-manager :as user-manager]
             [akvo.lumen.http :as http]
+            [akvo.lumen.lib.user :as user]
             [compojure.core :refer :all]))
 
 
-(defn endpoint [{:keys [tenant-manager user-manager]}]
-  (context "/api/admin/invites" {:keys [jwt-claims params server-name tenant]}
+(defn redirect-location
+  "Unfortually local development is done by running the api at port 3000
```
Comment (kardan) [inline backend/src/akvo/lumen/endpoint/invite.clj]: But from the users point of view they interact with our web app via one port 433. In our dev setup you need to jump between 3000& 3030. Makes sense?

File: backend/src/akvo/lumen/lib/user_impl.clj [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+  [keycloak tenant author-claims user-id]
+  (keycloak/demote-user-from-admin keycloak tenant author-claims user-id))
+
+(defn promote-user-to-admin
+  [keycloak tenant author-claims user-id]
+  (keycloak/promote-user-to-admin keycloak tenant author-claims user-id))
+
+
+
+;;;
```
Comment (kardan) [inline backend/src/akvo/lumen/lib/user_impl.clj]: Removing

File: backend/test/akvo/lumen/lib/user_test.clj [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;; System setup
+;;;
+
+(def seed-data
+  (->> "test-seed.edn" io/resource slurp edn/read-string))
+
+(def t1 (first (filter #(= "t1" (:label %))
```
Comment (kardan) [inline backend/test/akvo/lumen/lib/user_test.clj]: I was actually wondering if we even need the second tenant at all. Right now we have no tests or anything that use it.

File: backend/src/akvo/lumen/endpoint/invite.clj (hunk 1 above)
Comment (jonase) [inline backend/src/akvo/lumen/endpoint/invite.clj]: Could we not redirect `:3030/verify` in server.js similar to what we do with `:3030/share`.

**Pull request merged**